openkaarten-base.php

```diff
@@ -37,6 +37,14 @@
 	define( 'OWC_OPENKAARTEN_BASE_ASSETS_URL', esc_url( trailingslashit( plugins_url( '', __FILE__ ) ) . 'build' ) );
 }
 
+// Load Composer autoloader if available.
+if ( file_exists( __DIR__ . '/vendor/autoload.php' ) ) {
+	require_once __DIR__ . '/vendor/autoload.php';
+}
+
+// Initialize the OpenKaarten Base Functions class.
+Openkaarten_Base_Functions\Openkaarten_Base_Functions::init();
```

Comment:
Nog even een check of de class wel bestaat? Zonder een composer install zal hij namelijk nog niet bestaan en zal dit een fatal error geven.

Reply:
Ik heb een check toegevoegd overal in de code, ook in de Geodata plugin